pages/reference/supervisor/docker-compose.md

```diff
@@ -81,6 +80,7 @@ Field | Details
 [cpu_percent](https://docs.docker.com/compose/compose-file/compose-file-v2/#cpu-and-other-resources) |
 [cpus](https://docs.docker.com/compose/compose-file/compose-file-v2/#cpu-and-other-resources) | Introduced by Docker Compose v2.2 and therefore not working
 [env_file and service.env_file](https://docs.docker.com/compose/how-tos/environment-variables/set-environment-variables/#use-the-env_file-attribute)
+[expose](https://docs.docker.com/compose/compose-file/compose-file-v2/#expose) |
```

Comment:
Can you add something like "Informational only. Removed to prevent conflicts with other network configurations" on the details column?

Reply:
Done